test/framework/easyconfigs/yeb/intel-2015b.yeb

```diff
@@ -0,0 +1,22 @@
+easyblock: Toolchain
+
+name: intel
+version: 2015b
+
+homepage: 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description: Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MPI & Intel MKL.
+
+toolchain: {'name': 'dummy', 'version': 'dummy'}
+
+vars:
+    - &compver 2015.3.187
+    - &gccsuff -GNU-4.9.3-2.25
+
+dependencies: [
+    [icc, *compver, *gccsuff],
+    [ifort, *compver, *gccsuff],
+    [impi, 5.0.3.048, '', [iccifort, !join [*compver, *gccsuff]]],
```

Comment:
would `!join *compver, *gccsuff` also work? (less square brackets to worry about)